### ISSUE TITLE
Not getting any results

### ISSUE THREAD

**Issue by bslefevre** [mollie/mollie-api-node#20]
I've been using mollie-api-node for 2 years but now I'm not getting any results with the "payments.all()".

Using version "1.0.4" and now upgraded to "1.3.2".

Still not getting any results and didn't give "data" to "payments.all()" which I don't see why I need to do it and than request.write(JSON.stringify(data)); goes wrong.

Fixed that problem (if check and set "{}" to data) but still no results.

Is there something wrong with the servers?

Help!

**Comment by Thijs-Riezebeek** [comment]
Hi bslefevre,

Thank you for opening an issue.

If I understand you correctly, you are not seeing any payments when calling `payments.all()`?

The profile key that you use will only show payments that you have made on that profile. Furthermore, when using a test profile key (`test_...`) you will only see all of your test payments and vice versa for your live profile key.

If you do have payments on your profile, but are not seeing them by calling `payments.all()`, then something is going wrong.

Could you confirm for me that you do have payments on your profile but are not seeing them when you call `payments.all()`?

You can also send your customer ID in an email to me (thijs@mollie.com) and then I can investigate the issue myself. 

Also, you are correct in saying that you don't need to pass the `data` parameter to `payments.all()`. This is only used to set the `offset` and `count` API parameters.

**Comment by Thijs-Riezebeek** [comment]
Hi bslefevre,

Thank you for mailing me the details of your issue. I think I have found the cause of the issue you are describing.

We remove all test payments from the database that are more than 15 days old. This is probably why you are not seeing any of the test payments that you expected to see.

I've just created a test payment on your profile and when retrieving it via the API it did show up.

I hope this answers your question and resolved your issue. If not, feel free to leave another comment.

Kind regards,
Thijs Riezebeek
